### ISSUE TITLE
Message not marked as read when opened from Notifier

### ISSUE THREAD

**Issue by cmccandless** [kaypoz/reddit-notifier#14]
I.E. clicking on a message notification in the Notifier opens the message/comment link, but does not mark the message as 'Read'. 

For relevant API reference, see https://www.reddit.com/dev/api/#POST_api_read_message

**Comment by Wizek** [comment]
Yeah, this bothers me quite a bit too. Maybe it's easy to fix? Or maybe one of the forks have already fixed it?

**Comment by Wizek** [comment]
Looking briefly at the forks of @RodrigoZanotta, @adhesivecheese, and @DanTheMan827 it seems none of them have commits in this direction, mostly/only changing styling. Or please correct me if I've missed a relevant commit.

And at any rate, would one of you who've forked already be familiar with the codebase to tell us if this could be a simple change? Maybe just one extra XHR request on the click event handler to issue a mark-as-read command?

cc @kaypoz, @bgrawi, @goibon

**Comment by goibon** [comment]
Well there surely are some relevant lines of code [right here](https://github.com/kaypoz/reddit-notifier/blob/7c30c4c107e49098c67fd25bfaee33199bb57275/scripts/bg_fetch.js#L66-L78) that can be triggered by calling [this function here](https://github.com/kaypoz/reddit-notifier/blob/7c30c4c107e49098c67fd25bfaee33199bb57275/scripts/fetch.js#L77-L79).
It seems like it should be called when clicking a notification based on [these lines](https://github.com/kaypoz/reddit-notifier/blob/7c30c4c107e49098c67fd25bfaee33199bb57275/scripts/fetch.js#L155-L157) so my first guess is that it was the intention all along and something went wrong.
So yeah I think it's a fairly easy fix once you start diving into it and debugging.

Do you want to look into it and perhaps create a PR @Wizek ? 😃

**Comment by Wizek** [comment]
I just might. :) I'm glad to hear there is openness for a PR too; I always like to confirm this before starting working on something. @goibon If the code in the PR is to your liking, will you be able to merge it back and create a Web Store release too?

**Comment by goibon** [comment]
@Wizek I'd love to but you'll have to get the attention of either @kaypoz or @bgrawi I think 😕

**Comment by Wizek** [comment]
Alright, then let's hope one of them replies soon. Or if they don't, I guess the next best option is to release a fork ourselves with the fix, isn't it?

**Comment by bgrawi** [comment]
Hey all - I'd be happy to accept a PR that fixes this 😄 .
